rasterio/features.py

```diff
@@ -193,11 +193,10 @@ def rasterize(
 
     Parameters
     ----------
-    shapes : iterable of (`geometry`, `value`) pairs or iterable over
-        geometries. The `geometry` can either be an object that
-        implements the geo interface or GeoJSON-like object. If no
-        `value` is provided the `default_value` will be used. If `value`
-        is `None` the `fill` value will be used.
+    shapes : iterable of (`geometry`, `value`) pairs or iterable over geometries
+        The `geometry` can either be an object that implements the geo interface
+        or GeoJSON-like object. If no `value` is provided the `default_value`
+        will be used. If `value` is `None` the `fill` value will be used.
```

Comment:
I prefer to keep docstrings to 72 chars.

Reply:
So the reason for this change is that, as far as I can tell, the "type" of a parameter must fit on a single line in numpy docstring style. Otherwise you end up with docs looking like https://rasterio.readthedocs.io/en/latest/api/rasterio.features.html#rasterio.features.rasterize

Let me try to make the type description shorter.

Reply:
P.S. I use flake8 and black to enforce line length endings and it's really helped my projects!